File: Modul 3/SOAL 5/PRAK205-2210817210002-GustiMuhammadNaufalRazin.py
import cmath
a2,b2 = input().split()
a2 = int(a2)
b2 = int(b2)
c2 = (b2*b2-a2*a2) ** (1/2)
k2 = a2+b2+c2
l2 = 0.5*c2*a2
print("Alas = %.f cm"%c2)
print("Tinggi = %.f cm"%a2)
print("Keliling = %.f cm"%k2)
print("Luas = %.f cm^2"%l2)

a = int(input(""))
b = int(input(""))
c = (b*b-a*a) ** (1/2)
k = a+b+c
l = 0.5*c*a
print("Alas = %.f cm"%c)
print("Tinggi = %.f cm"%a)
print("Keliling = %.f cm"%k)
print("Luas = %.f cm^2"%l)

File: Modul 3/SOAL 5/PRAK205-2210817210002-GustiMuhammadNaufalRazin.c
#include <stdio.h>
int main()
{
    int i = 0;
    while(i<=1){
    int a,b,c,k,l;
    scanf("%d %d",&a,&b);
    c = sqrt(b*b-a*a);
    k = a+b+c;
    l = 0.5*c*a;
    printf("Alas = %d cm\n",c);
    printf("Tinggi = %d cm\n",a);
    printf("Keliling = %d cm\n",k);
    printf("Luas = %d cm^2\n\n",l);
    i++;
}
}
